File: src/main/java/project/xml/a1/IzvornoDelo.java
package project.xml.a1;

import lombok.Getter;
import lombok.Setter;

import javax.xml.bind.annotation.*;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "", propOrder = {
    "naslov",
    "autor",
    "anonimanAutor"
})
@XmlRootElement(name = "Izvorno_delo")
@Getter
@Setter
public class IzvornoDelo {

    @XmlElement(name = "Naslov", required = true)
    protected String naslov;
    @XmlElement(name = "Autor")
    protected TAutor autor;
    @XmlElement(name = "Anoniman_autor", defaultValue = "true")
    protected Boolean anonimanAutor;

}

File: XML_client/src/app/a1/model/izvorno-delo.ts
import {TZivAutor} from "./t-ziv-autor";
import {TPreminuliAutor} from "./t-preminuli-autor";
import {TAutor} from "./t-autor";

export class IzvornoDelo{

  constructor(
    public naslov: string = '',
    public autor: TAutor = new TAutor(),
    public anonimanAutor: boolean = false
  ) {}


}
